pagopa/io-app
# fix: [IOBP-84] Fix visual glitches on Android devices

## Short description
This PR fixes visual glitches on Android devices in IDPay screens.

## List of changes proposed in this pull request
- [ts/components/core/icons/svg/IconIoAppBlueBackground.tsx](https://github.com/pagopa/io-app/pull/4686/files#diff-be49b25519dc58654607509a01e652550612d663b69e725bc1755cd3af893e3b): 
     - Added SVG size props
- [ts/features/idpay/initiative/details/components/InitiativeDetailsBaseScreenComponent.tsx](https://github.com/pagopa/io-app/pull/4686/files#diff-a572623e06de55ed1f5b04ac7f6652a612c500ca7fd00b72dd24e0a42eb1ed30): 
    - Added `StatusBar` `backgroundColor` and  `barStyle` props
    - Fixed footer button padding
 - [ts/features/idpay/initiative/details/screens/InitiativeDetailsScreen.tsx](https://github.com/pagopa/io-app/pull/4686/files#diff-f7ee97294ece37f95074427629a1371d562c1362a66ce188ad0a7a7f5a4ecd37): 
     - Removed footer button padding
- [ts/features/idpay/initiative/timeline/components/TimelineDetailsBottomSheet.tsx](https://github.com/pagopa/io-app/pull/4686/files#diff-9563abc84afa7963c9e7003f0a06e705c88a054a17493935ab917e899dd9d135)
     - Fixed bottom sheet footer padding
     
## How to test
With the `io-dev-server`, navigate to the IDPay screens on an Android and iOS devices/emulators. Check that bot UI are the same.
